packages/cms-data-sync/test/teams/teams.data-migration.test.ts

```diff
@@ -0,0 +1,316 @@
+            tools: {
+              'en-US': [
+                {
+                  sys: {
+                    id: 'entry-id',
+                    linkType: 'Entry',
+                    type: 'Link',
+                  },
+                },
+              ],
+            },
+          },
+        },
+      );
+    });
+
+    it('for a team that contains tools with empty description', async () => {
+      const toolWithEmptyDescription = {
+        name: 'Drive folder',
+        description: '',
+        url: 'http://www.google-drive.com/123',
+      };
+      const teamWithToolWithEmptyDescription = getTeamSquidexResponse();
+      teamWithToolWithEmptyDescription.flatData.tools = [
+        toolWithEmptyDescription,
+      ];
+
+      squidexGraphqlClientMock.request.mockResolvedValueOnce({
+        queryTeamsContents: [teamWithToolWithEmptyDescription],
+      });
+
+      const toolMock = getEntry({});
+      toolMock.publish = jest.fn().mockResolvedValueOnce(getEntry({}));
+      contenfulEnv.createEntry = jest.fn().mockResolvedValueOnce(toolMock);
+
+      await migrateTeams();
+
+      expect(contenfulEnv.createEntry).toHaveBeenCalledWith('externalTools', {
+        fields: {
+          name: { 'en-US': toolWithEmptyDescription.name },
+          url: { 'en-US': toolWithEmptyDescription.url },
+        },
+      });
+      expect(toolMock.publish).toBeCalled();
+
+      expect(contenfulEnv.createEntryWithId).toHaveBeenCalledWith(
+        'teams',
+        'team-1',
+        {
+          fields: {
+            displayName: { 'en-US': 'Team ASAP' },
+            applicationNumber: { 'en-US': '2023' },
+            expertiseAndResourceTags: { 'en-US': [] },
+            inactiveSince: { 'en-US': null },
+            projectSummary: { 'en-US': null },
+            projectTitle: { 'en-US': 'Beautiful Title' },
+            tools: {
+              'en-US': [
+                {
+                  sys: {
+                    id: 'entry-id',
+                    linkType: 'Entry',
+                    type: 'Link',
+                  },
+                },
+              ],
+            },
+          },
+        },
+      );
+    });
+  });
+
+  it('outputs an error message when trying to input a tool with invalid link', async () => {
+    const tool = {
+      name: 'Invalid link',
+      description: 'It contains a not valid url',
+      url: '123',
+    };
+    const teamWithInvalidURLTool = getTeamSquidexResponse();
+    teamWithInvalidURLTool.id = 'team-invalid-url-tool';
+    teamWithInvalidURLTool.flatData.tools = [tool];
+
+    squidexGraphqlClientMock.request.mockResolvedValueOnce({
+      queryTeamsContents: [teamWithInvalidURLTool],
+    });
+
+    await migrateTeams();
+
+    expect(console.log).toHaveBeenCalledWith(
+      '\x1b[31m',
+      '[ERROR] Invalid tool URL linked to team with id: team-invalid-url-tool',
+    );
+  });
+
+  it('still creates the team even if there is a tool with invalid link', async () => {
```

Comment:
can you add another test for when there are 2 tools one of which has an invalid link?